assets/component-localization-form.css

```diff
@@ -182,8 +329,16 @@ noscript .localization-selector.link {
   color: rgb(var(--color-foreground));
 }
 
-.disclosure__link--active {
-  text-decoration: underline;
+.disclosure__link .icon-checkmark {
+  width: 1rem;
+  margin-right: 0.2rem;
```

Comment:
This is working for the country selection but not language. We should be adding the same icon to the language picker otherwise it's hard to tell what is currently selected: 
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/13-14-15fch-gy8kb.png)
</details>

Reply:
Good catch. Thank you!